spec/acceptance/wget_system_spec.rb

```diff
@@ -1,7 +1,7 @@
 require 'spec_helper_acceptance'
 
 describe 'wget' do
-  let(:wget_manifest) { "class { 'wget': }" }
+  let(:wget_manifest) { "user { 'vagrant': ensure => present, managehome => true } -> class { 'wget': }" }
```

Comment:
I'd keep this local to the user test since it's not needed as part of the other tests.